api/v1/helpers/verbs/utils.js

```diff
@@ -565,7 +565,7 @@ function cleanAndStripNulls(obj) {
 
       // if undefined, parentAbsolutePath needs to be set to empty string,
       // to pass swagger's schema validation
-      if (key === 'parentAbsolutePath' && !o[key]) {
+      if ((key === 'parentAbsolutePath' || key === 'createdBy') && !o[key]) {
```

Comment:
To clarify, do we want to have a createdBy field when there is a token provided, otherwise not return a createdAt field? That seems a bit inconsistent. Don't we always want to return a createdAt field, similar to how we always want a parentAbsolutePath, or parentId?